src/passes/analysis/verilog/inline_utils.cpp

```diff
@@ -113,6 +113,11 @@ std::unique_ptr<vAST::StructuralStatement> inline_binary_op(
     transformer.visit(std::move(primitive_expr)));
 }
 
+bool is_mantle_wire(CoreIR::Module* module) {
```

Comment:
fyi: we mix snake_case and camelCase functions, I think the LLVM standard followed more frequently in this code base is camelCase (though most functions in this file seem to be snake_case 😣)

Reply:
Yea, that's annoying.  I'm fine with moving to camelCase, but maybe let's separate that change into a style only changeset, but I can follow that for future code.